src/core/importType.js

```diff
@@ -13,8 +13,9 @@ export function isAbsolute(name) {
 }
 
 export function isBuiltIn(name, settings) {
+  const baseModule = name.split('/')[0]
```

Comment:
This logic already exists in the project: https://github.com/benmosher/eslint-plugin-import/blob/ee5a986cfd3ed9d97857889655a969031f65f66d/src/core/importType.js#L31-L33 (and it's best to avoid directly indexing into arrays)

Something like this could be good:
```js
function baseModule(name) {
  if (isScoped(name)) {
    const [scope, pkg] = name.split('/');
    return `${scope}/${pkg}`;
  }
  const [pkg] = name.split('/');
  return pkg;
}
```